### PR TITLE
REF explicit urls

### PR DESCRIPTION
**Pull request by beckermr**
Checklist
* [x] Used a [fork of the feedstock to propose changes](https://conda-forge.org/docs/maintainer/updating_pkgs.html#forking-and-pull-requests)
* [x] Bumped the build number (if the version is unchanged)
* [ ] Reset the build number to `0` (if the version changed)
* [x] [Re-rendered]( https://conda-forge.org/docs/maintainer/updating_pkgs.html#rerendering-feedstocks ) with the latest `conda-smithy` (Use the phrase <code>@<space/>conda-forge-admin, please rerender</code> in a comment in this PR for automated rerendering)
* [x] Ensured the license file is being packaged.

### CODE REVIEW

**Comment by beckermr** [comment]
@conda-forge-admin rerender

**Comment by beckermr** [comment]
@conda-forge-admin rerender

**Comment by beckermr** [comment]
@ocefpaf this version of the recipe should work with the bot

**Comment by beckermr** [comment]
@conda-forge/jaxlib this one is ready for review!

File: recipe/meta.yaml [hunk 1]
```diff
@@ -1,33 +1,46 @@
...
   name: jaxlib
   version: {{ version }}
 
 source:
-  fn: {{ fn }}
-  url: https://pypi.io/packages/{{ pyver }}/{{ name[0] }}/{{ name }}/{{ fn }}
-  sha256: 74490dd3c2e37492dbefc040a059bdedf0ba62d5d68c51d43181ac6c1aa14518  # [linux and py==36]
+  fn: wheel.whl
+
+  url: https://pypi.io/packages/cp36/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}-cp36-none-{{ linux_whl }}.whl  # [linux and py==36]
```
Comment (ocefpaf) [inline recipe/meta.yaml:23]: @ericmjl this is quite ugly and redundant but it is the only way to make the new bot work here.

Maybe it is time to work on a source compilation again :smile:

Comment (beckermr) [reply recipe/meta.yaml:23]: We might be able to improve the bot OFC, but no time for that right now.

**Pull request merged**